src/state/ducks/student/reducer.ts

```diff
@@ -1,10 +1,44 @@
 import { createSlice } from "@reduxjs/toolkit";
-
+import { IStudentRaw } from "./types";
 export const userSlice = createSlice({
   name: "student",
-  initialState: { records: [] },
+  initialState: {
+    data: [
+      {
+        _id: 1,
+        name: "Javed",
+        marks: 100,
+        subject: "Math",
+        grade: "A+",
+        time: "2022-07-18T08:07:28.3Z",
+      },
+      {
+        _id: 2,
+        name: "Ahmad",
+        marks: 80,
+        subject: "English",
+        grade: "B+",
+        time: "2022-07-18T08:07:28.3Z",
+      },
+      {
+        _id: 3,
+        name: "zakria",
+        marks: 40,
+        subject: "English",
+        grade: "F",
+        time: "2022-07-18T08:07:28.3Z",
+      },
+    ],
+  },
+
   reducers: {
+    FETCH_STUDENTS_SUCCESS: (
```

Comment:
**FETCH_STUDENTS_SUCCESS** fells like a const variable rathar then a function nam.
change this name to **fetchStudentsSuccess** and make ajustment according everywhere.